akka-stream-tests/src/test/scala/akka/stream/scaladsl/SinkForeachParallelSpec.scala

```diff
@@ -82,13 +82,11 @@ class SinkForeachParallelSpec extends AkkaSpec {
           Await.ready(latch, 10.seconds)
         }
       }).withAttributes(supervisionStrategy(resumingDecider)))
-      p.onFailure { case e ⇒ assert(e.getMessage.equals("err1")); Unit }
 
       latch.countDown()
       probe.expectMsgAllOf(1, 2, 4)
 
       Await.result(p, 5.seconds)
-      assert(p.isCompleted)
```

Comment:
That's a nonsense-assert after waiting for the result. And we don't care about the result itself unless it's an exception, which would indicate failure, and would be rethrown by Await.